incubator/exporter-otlp/src/test/java/io/aklivity/zilla/runtime/exporter/otlp/internal/OtlpExporterTaskTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.otlp.internal;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+
+import io.aklivity.zilla.runtime.exporter.otlp.internal.serializer.OtlpMetricsSerializer;
+
+public class OtlpExporterTaskTest
+{
+    @Test
+    public void shouldPost()
+    {
+        // GIVEN
+        OtlpMetricsSerializer serializer = mock(OtlpMetricsSerializer.class);
+        when(serializer.serializeAll()).thenReturn("json");
+        OtlpExporterTask task = new OtlpExporterTask("http://example.com", serializer);
+
+        // WHEN
+        task.run();
+
+        // THEN
+        verify(serializer).serializeAll();
+    }
+}
```

Comment:
This unit test should be replaced with a `k3po` integration test that verifies the output produced via `http` request using open telemetry syntax.

Reply:
I will work on the IT & will delete this.

Reply:
pushed the IT, deleted this